README.md

```diff
@@ -455,12 +469,7 @@ network_connections:
       auto6: no
 ```
 
-Like for `master`, the `parent` references the connection profile in the ansible
-role.
-
-### `type: macvlan`
-
-MACVLANs also work:
+Configuring macvlan:
```

Comment:
Since VLAN is all upper-case, maybe it should be MACVLAN as well or at least MacVLAN.